### DISCUSSION TITLE
too many hiveserver2 connection

### DISCUSSION THREAD

**Discussion by forest455** [apache/dolphinscheduler#12213] [Q&A]
![IMG_20220928_194714_edit_273430840399422](https://user-images.githubusercontent.com/95460060/193016587-798c43a4-649c-4bd5-b0c5-4346b8b8a228.jpg)
I found no workflow running, but why worker keep printing hive connecting logs.

**Comment by DarkAssassinator** [comment]
please check this PR: https://github.com/apache/dolphinscheduler/pull/12226

**Comment by forest455** [comment]
Thanks

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
